OpenOversight/app/models/config.py

```diff
@@ -52,7 +59,14 @@ def __init__(self):
         self.OO_SERVICE_EMAIL = os.environ.get(KEY_OO_SERVICE_EMAIL)
         # TODO: Remove the default once we are able to update the production .env file
         # TODO: Once that is done, we can re-alpha sort these variables.
-        self.OO_HELP_EMAIL = os.environ.get("OO_HELP_EMAIL", self.OO_SERVICE_EMAIL)
+        self.OO_HELP_EMAIL = os.environ.get(KEY_OO_HELP_EMAIL, self.OO_SERVICE_EMAIL)
+
+        # Flask-Mail-related settings
+        setattr(self, KEY_MAIL_SERVER, os.environ.get(KEY_MAIL_SERVER))
+        setattr(self, KEY_MAIL_PORT, os.environ.get(KEY_MAIL_PORT))
+        setattr(self, KEY_MAIL_USE_TLS, str_is_true(os.environ.get(KEY_MAIL_USE_TLS)))
+        setattr(self, KEY_MAIL_USERNAME, os.environ.get(KEY_MAIL_USERNAME))
+        setattr(self, KEY_MAIL_PASSWORD, os.environ.get(KEY_MAIL_PASSWORD))
```

Comment:
What's the reasoning for using `setattr` instead of `self.____`?

Reply:
Yea, I was going back and forth about using `setattr` here but this lets us use the value of the constant as the config variable name, rather than hardcoding that name (for example `self.MAIL_SERVER = ...`) because otherwise we would need to update config.py along with constants.py if we wanted to change the config variable name.

I don't have a strong preference either way so let me know if you want me to change it

Reply:
The more I think about it, the more I like it. Keep it, imo. I may change the syntax for the rest of them in another PR.